common/api-review/storage.api.md

```diff
@@ -158,6 +217,32 @@ export interface UploadTask {
     then(onFulfilled?: ((snapshot: UploadTaskSnapshot) => unknown) | null, onRejected?: ((error: FirebaseStorageError) => unknown) | null): Promise<unknown>;
 }
 
+// @internal
+export class _UploadTask {
+    constructor(ref: _Reference, blob: _FbsBlob, metadata?: Metadata | null);
+    _blob: _FbsBlob;
+    cancel(): boolean;
+    catch<T>(onRejected: (p1: FirebaseStorageError_2) => T | Promise<T>): Promise<T>;
+    // Warning: (ae-forgotten-export) The symbol "Metadata" needs to be exported by the entry point index.d.ts
+    _metadata: Metadata | null;
+    // Warning: (ae-forgotten-export) The symbol "TaskEvent" needs to be exported by the entry point index.d.ts

```

Comment:
Not really because `_UploadTask` is `@internal` and only used by `storage-compat`, so we know everything works fine.
If it is a public symbol, we should address these warnings.